cel/options.go

```diff
@@ -156,26 +156,25 @@ func Container(name string) EnvOption {
 //    // Simplified Object construction
 //    ObjTypeName{field: FieldTypeName{value: ...}}
 //
-// There are a few rules for the qualified names and the simple name abbreviations generated from
-// them:
+// There are a few rules for the qualified names and the simple abbreviations generated from them:
 // - Qualified names must be dot-delimited, e.g. `package.subpkg.name`.
-// - The last element in the qualified name is the simple name used as an abbreviation.
-// - Abbreviations must not collide with each other.
-// - The abbreviation must not collide with the first component of a qualified container name.
+// - The last element in the qualified name is the simple name used as the abbreviation.
+// - Abbreviations names must not collide with each other.
+// - The abbreviation must not collide with unqualified names in use.
 //
-// Abbreviations are distinct from container-based references in the following important ways:
+// Abbrevs are distinct from container-based references in the following important ways:
 // - Containers follow C++ namespace resolution rules with searches from the most qualified name
```

Comment:
See if the comment flows better moving the first two items (lines 166-169) to the end of the group - discuss abbrevs first because they are resolved first.